src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_lord_jaraxxus.cpp

```diff
@@ -21,6 +21,7 @@
 #include "SpellScript.h"
 #include "Player.h"
 #include "trial_of_the_crusader.h"
+#include <math.h>
```

Comment:
@Keader ignore this
@Naios They very much are standard both in C and C++ - it is recommended to use <cmath> (C doesnt have <cxxxxxx>)